e2e/test/scenarios/embedding/sdk-iframe-embedding-setup/select-embed-experience.cy.spec.ts

```diff
@@ -161,24 +161,24 @@ H.describeWithSnowplow(suiteTitle, () => {
     );
   });
 
-  // TODO: fix this flaky test
-  it(
-    "localizes the iframe preview when ?locale is passed",
-    { tags: "@skip" },
-    () => {
-      visitNewEmbedPage({ locale: "fr" });
-
-      // TODO: update this test once "Exploration" is localized in french.
-      getEmbedSidebar().findByText("Exploration").click();
+  it("localizes the iframe preview when ?locale is passed", () => {
+    visitNewEmbedPage({ locale: "fr" });
 
-      H.waitForSimpleEmbedIframesToLoad();
+    // TODO: update this test once "Exploration" is localized in french.
+    getEmbedSidebar().findByText("Exploration").click();
 
-      H.getSimpleEmbedIframeContent().within(() => {
-        cy.log("data picker is localized");
-        cy.findByText("Choisissez vos données de départ").should("be.visible");
-      });
-    },
-  );
+    H.waitForSimpleEmbedIframesToLoad();
+
+    H.getSimpleEmbedIframeContent().within(() => {
+      cy.log("data picker is localized");
+
+      cy.findByText("Données", { timeout: 10_000 }).should("be.visible");
```

Comment:
It takes a bit of time before the iframe content (i.e. SDK) loads, so we need an extra timeout to reduce flakiness on CI.

Reply:
does this extra assertion necessary or could we just rely on the only existing one assertion but set the timeout to 10s?